src/sentry/testutils/skips.py

```diff
@@ -67,11 +69,23 @@ def _requires_snuba() -> None:
 @pytest.fixture(scope="session")
 def _requires_kafka() -> None:
     kafka_conf = settings.SENTRY_DEVSERVICES["kafka"](settings, {})
+    zk_conf = settings.SENTRY_DEVSERVICES["zookeeper"](settings, {})
     (port,) = kafka_conf["ports"].values()
 
     if not _service_available("127.0.0.1", port):
         pytest.skip("requires kafka server running")
 
+    try:
+        check_health(
+            "kafka",
+            {
+                "kafka": kafka_conf,
+                "zookeeper": zk_conf,
+            },
+        )
+    except Exception as e:
+        pytest.skip(f"kafka server is not heathy: {e}")
```

Comment:
Could this also mean zookeeper isn't healthy?

Reply:
Yeah this health check technically requires kafka and zookeeper.

I intend to add a separate healthcheck for zookeeper and I'd like to change this check to be just for kafka.

Reply:
I'm actually going to remove this change, so I can separate out the addition of healthchecks from issues with the kafka healthcheck (i.e. reverting the change will be less disruptive.)